doc/languages-frameworks/r.section.md

```diff
@@ -100,19 +100,23 @@ re-enter the shell.
 nix-shell generate-shell.nix
 
 Rscript generate-r-packages.R cran  > cran-packages.nix.new
-mv cran-packages.nix.new cran-packages.nix
-
 Rscript generate-r-packages.R bioc  > bioc-packages.nix.new
-mv bioc-packages.nix.new bioc-packages.nix
-
 Rscript generate-r-packages.R bioc-annotation > bioc-annotation-packages.nix.new
-mv bioc-annotation-packages.nix.new bioc-annotation-packages.nix
-
 Rscript generate-r-packages.R bioc-experiment > bioc-experiment-packages.nix.new
+
+sleep 1
```

Comment:
Which OS/filesystem has the locking issues? This doesn't sound like it's POSIX compliant, once the process exits the descriptors should be closed. I don't think we should be working around this by delaying an arbitrary amount.

Reply:
I use NixOS in a virtualbox VM, and I have cloned nixpkgs into a virtualbox shared folder. The shared folder points to an underlying NTFS formatted drive.

If that combination is not worth supporting, then I can remove this.

Reply:
I think that explains it, it's common for shared/distributed filesystems to not be fully POSIX compliant and race conditions like this can arise. This is difficult to handle as even waiting 1s may be insufficient for some filesystems. I'd rather not include this.

Reply:
Sure, I will remove it.